variables.tf

```diff
@@ -215,6 +215,54 @@ EOF
 # TLS
 # --------------------
 
+variable user_managed_tls {
+  type = "string"
+  default = false
+  description = <<EOF
+Set to true if you'd like to manage and upload your own TLS files, if you do not want this module
+to generate them. By default this module expects the following files at the root of the bucket, but these
+can be overriden:
+- `ca.crt`: Root CA public certificate
+- `vault.crt`: Vault server public certificate, signed by the ca.crt
+- `vault.key.enc` Vault server certificate private key, encrypted with the kms key provided and base64 encoded.
+EOF
+}
+
+variable vault_tls_bucket {
+  type = "string"
+  default = ""
+  description = <<EOF
+GCS Bucket override where Vault will expect TLS certificates are stored.
+EOF
+}
+
+variable vault_ca_cert_filename {
+  type = "string"
+  default = "ca.crt"
+  description = <<EOF
+GCS object path within the vault_tls_bucket. This is the root CA certificate.
+Default: ca.crt
```

Comment:
You don't need to mark defaults in the description (it will show up automatically in the output unless it's a complex one)